### PR TITLE
LPS-193662 | Test Fix | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-193662

cc: @patriciajeanperez

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/cpdynamicdatalists/CPDynamicdatalists.testcase [hunk 1]
```diff
@@ -1124,90 +1124,95 @@ definition {
 	@description = "This is a use case for LPS-65726"
 	@priority = 5
 	test AddListRecordWithLinkToPageField {
-		ProductMenu.gotoPortlet(
-			category = "Site Builder",
-			portlet = "Pages");
+		task ("Given a list with a Link to page field") {
```
Comment (kristinonias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/cpdynamicdatalists/CPDynamicdatalists.testcase:1127]: Added task blocks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/cpdynamicdatalists/CPDynamicdatalists.testcase [hunk 2]
```diff
@@ -1124,90 +1124,95 @@ definition {
...
+			DDLRecord.populateFieldLinkToPage(
+				ddlRecordFieldData = "Test Page",
+				fieldName = "LinkToPage",
+				pagePrivacy = "Public");
 
-		Click(
-			key_indexNumber = 1,
-			key_text = "Publish",
-			locator1 = "Button#ANY_N");
+			SelectFrameTop();
```
Comment (kristinonias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/cpdynamicdatalists/CPDynamicdatalists.testcase:1193]: Added this to ensure that the publish button can be clicked.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**